library/axi_clkgen/axi_clkgen_ip.tcl

```diff
@@ -17,6 +17,12 @@ adi_ip_files axi_clkgen [list \
   "bd/bd.tcl" \
   "axi_clkgen.v" ]
 
+set_property used_in_simulation false [get_files ./bd/bd.tcl]
+set_property used_in_simulation false [get_files  ./../scripts/adi_xilinx_device_info_enc.tcl]
+set_property used_in_synthesis false [get_files ./bd/bd.tcl]
+set_property used_in_synthesis false [get_files ./../scripts/adi_xilinx_device_info_enc.tcl]
```

Comment:
Wouldn't it be better if it was written with the $ad_hdl_dir, since it anyway is used at line 8?
```suggestion
set_property used_in_simulation false [get_files $ad_hdl_dir/library/scripts/adi_xilinx_device_info_enc.tcl]
set_property used_in_synthesis false [get_files ./bd/bd.tcl]
set_property used_in_synthesis false [get_files $ad_hdl_dir/library/scripts/adi_xilinx_device_info_enc.tcl]
```